packages/helpers/src/index.ts

```diff
@@ -14,3 +14,4 @@ export * from "./checkout";
 export * from "./languages";
 export * from "./relativeUrlSlash";
 export * from "./urlIsAbsolute";
+export * from "./b2bQuote Management/canUseQuoteActions";
```

Comment:
Why do we have a space here? 
Looks strange also in `helpers/src/b2bQuote Management`.
I would suggest removing the space.

Reply:
by mistake, fixed